path_to_folder.py

```diff
@@ -0,0 +1,17 @@
+# region path to a folder-this is the way
+import os
+
+
+def path_to_folder():
+    """function that receives a path to a folder,
+    and returns the list of all files that start with the sequence of letters "deep" in that folder.
+    """
+    path = input('Enter path to folder: ')
+    list_of_files = os.listdir(path)
+    for file in list_of_files:

```

Comment:
1.  You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs. 

2.Seprate IO from logic - 
https://wiki.c2.com/?SeparateIoFromCalculation

Reply:
ok!
thanks, i want to change the code. what the best way to do it?